src/ExternalTrackerGoogleAnalyticsBundle/Bridge/Chameleon/Migration/Script/update-1650875276.inc.php

```diff
@@ -0,0 +1,25 @@
+<h1>Build #1650875276</h1>
+<h2>Date: 2022-04-25</h2>
+<div class="changelog">
+    - Add GA4 based tracker
+</div>
+<?php
+
+$data = TCMSLogChange::createMigrationQueryData('pkg_external_tracker', 'en')
+  ->setFields([
+      'class_subtype' => '',
+      'class_type' => 'Customer',
+      'id' => '44922d95-4182-19c6-68c7-9d99ae23d57a',
+      'name' => 'Google Analytics GA4',
+      'active' => '0',
+      'identifier' => '',
+      'test_identifier' => '',
+      'class' => 'ChameleonSystem\ExternalTrackerGoogleAnalyticsBundle\Bridge\Chameleon\ExternalTracker\ExternalTrackerGoogleAnalyticsGa4',
+  ]);
+TCMSLogChange::insert(__LINE__, $data);
+
+$data = TCMSLogChange::createMigrationQueryData('pkg_external_tracker', 'de')
+  ->setFields([ 'name' => 'Google Universal Analytics (analytics.js, alt)' ])
+  ->setWhereEquals([ 'id' => '7ba2f2ab-6426-6e13-443e-7bde372406cd' ]);
```

Comment:
This has a varying id in all projects.

Reply:
Oh, I didn't realize that.

:heavy_check_mark:  I updated the migration to select by `class`, which should always be the same instead.